test/api/v3/integration/tasks/POST-tasks_unlink-one_taskId.test.js

```diff
@@ -92,16 +92,16 @@ describe('POST /tasks/unlink-one/:taskId', () => {
 
   it('unlinks a task from a challenge and saves it on keep=keep', async () => {
     await user.post(`/tasks/challenge/${challenge._id}`, tasksToTest.daily);
-    let [, daily] = await user.get('/tasks/user');
```

Comment:
all these changes from `[, task]` to `[task]` are because new tasks are now pushed to the top of the list, right?

Reply:
It's because previously, I think the tasks were sorted by the order in which they were added - since the user was generated with a Todo, that Todo originally came first. Now, however, this sorts the challenges according to task order. Since it wasn't clear how to return the different task types when we ask for all the types at the same time, I chose the order `habits`, `dailies`, `todos`, and `rewards` (the same way they're displayed on the tasks page). As such, the daily now comes before the Todo, and I've updated the call to reflect that.